packages/compartment-mapper/src/policy.js

```diff
@@ -70,6 +70,34 @@ const generateCanonicalName = ({ isEntry = false, name, path }) => {
   return path.join('>');
 };
 
+const POLICY_FIELDS = ['builtins', 'globals', 'packages'];
+
+/**
+ *
+ * @param {Object} packagePolicy
+ * @param {string} field
+ * @param {string} itemName
+ * @returns {boolean|Object}
+ */
+const policyLookupHelper = (packagePolicy, field, itemName) => {
+  if (!POLICY_FIELDS.includes(field)) {
+    throw Error(`Invalid field ${q(field)}`);
+  }
+  if (typeof packagePolicy !== 'object' || packagePolicy === null) {
+    return false;
+  }
+
+  if (packagePolicy[field]) {
+    if (packagePolicy[field] === 'any') {
+      return true;
+    }
+    if (packagePolicy[field][itemName]) {
+      return packagePolicy[field][itemName];
+    }
+  }
```

Comment:
I'd rather see this broken out.  I'll defer to your judgement, though:
```suggestion
  if (packagePolicy[field] === 'any') {
    return true;
  }
  if (packagePolicy[field]) {
    if (packagePolicy[field][itemName]) {
      return packagePolicy[field][itemName];
    }
  }
```

Reply:
I need to make sure packagePolicy[field] exists first. I can make it an early return maybe.

Reply:
:+1: